scripts/helpers/networkid.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(web3) {
+  return new Promise((resolve, reject) => {
+    web3.version.getNetwork((err, network) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(network);
```

Comment:
Do the address files need the `networkId` as a string? Otherwise I would do the `parseInt` here.

Reply:
yeah... that keeps bothering me also. but I don't know... maybe we sometimes use strings maybe integers. 
I think I should go through the whole code once and make sure we use strings everywhere. 
but also I am nut sure what web3 or ethers.js return as network id - if that is consistent.